src/cubicbez.rs

```diff
@@ -348,6 +348,26 @@ impl CubicBez {
             .filter(|t| *t >= 0.0 && *t <= 1.0)
             .collect()
     }
+
+    /// Find lines from the point to the curve tangent to the curve.
```

Comment:
On my initial few read-throughs of this sentence I had a hard time groking it. I can also take a shot at wording it, which might also be flawed, but maybe we can meet in the middle with something that's clearer to unbiased readers:

> Find points on the curve where each one's tangent line passes through the given point in space.

Reply:
I made it a little shorter (it should be one line), but agree this is clearer. I think in my mind I was thinking about the line from the point to the curve, but thinking about it the other way is simpler because it's more apparent that you're getting both the tangent and the incidence.